### ISSUE TITLE
Feature: Use Azure Translation Text API for translations?

### ISSUE THREAD

**Issue by michael-nee** [simplcommerce/SimplCommerce#529]
When I get time I'll have a look at this:

https://www.productivecsharp.com/2018/04/translator-text-api/

**Comment by hishamco** [comment]
We already implemented an `EFStringLocalizer` but the translation may useful if we use caching

**Comment by thiennn** [comment]
I think @michael-nee might thinking about translate dynamic content like product name, product description,.... Our localization only support static content like labels

**Comment by michael-nee** [comment]
> I think @michael-nee might thinking about translate dynamic content like product name, product description,.... Our localization only support static content like labels

Yes that's correct, but just wondering how it could be used dynamically.

**Comment by hishamco** [comment]
This is can be happened on fly, and it would be nice if create a hook for content loaded, so you can subscribe and translate the content when the event fires
